### DISCUSSION TITLE
if i disable video playback as such it improves performance?

### DISCUSSION THREAD

**Discussion by grumi123** [pear-devs/pear-desktop#765] [Q&A]
my pc is not that good and sometimes it goes a little slow if i disable video playback it improves its perfrormace?

**Comment by RaveVR** [comment]
good question is how to disable video playback, i had set it before, but now after i restarted my PC and installed YT Music again i cant find it

**Reply by Araxeus** [reply]
There is a `video-toggle` plugin

**Reply by RaveVR** [reply]
aaaah sh1t, i always had on minimalized, and didnt saw that plugin, i set on fullscreen and now i can see that, thanks

**Reply by Araxeus** [reply]
you're using `in-app-menu` right? 
the problem with that plugin is that the menu is cut when the window size is small :(
